ConnectorArchicad/ConnectorArchicad/Communication/AsyncCommandProcessor.cs

```diff
@@ -27,7 +27,15 @@ internal class AsyncCommandProcessor
     {
       return Task.Run(command.Execute, token);
     }
-    catch (Exception e)
+    catch (ArgumentNullException)
+    {
+      return null;
+    }
+    catch (TaskCanceledException)
+    {
+      return null;
+    }
+    catch (ObjectDisposedException)
```

Comment:
I'm assuming `Execute` cannot throw within? If so, would we want to add logging to some these since we're swallowing them?